pom.xml

```diff
@@ -20,6 +20,7 @@
             </activation>
             <properties>
                 <cloudhsmJarPath>/opt/cloudhsm/java/cloudhsm-2.0.0.jar</cloudhsmJarPath>
+                <cloudhsmVersion>2.0.0</cloudhsmVersion>
```

Comment:
May as well bump this to 2.0.3 as that is the latest in our repo.

Reply:
Done